spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'web_helpers'
+
+RSpec.feature "Sessions",  type: :feature do
+
+  scenario "signs up the user", :type => :feature do
+    sign_up
+    page.should have_content("Welcome! You have signed up successfully.")
+  end
+
+  scenario "signs in the user", :type => :feature do
+    sign_up
+    visit "/users/sign_out"
+    sign_in
+    page.should have_content("Signed in successfully.")
+  end
+
+  scenario "signs in the user", :type => :feature do
+    sign_up
+    visit "/users/sign_out"
+    page.should have_content("Signed out successfully.")
+  end
+end
```

Comment:
Jess and I noticed that there are some duplicates/unmatching tests here, could you please fix these(Sign in and sign out tests)?